packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.tsx

```diff
@@ -3,6 +3,7 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import type { IModelConnection } from "@itwin/core-frontend";
```

Comment:
The length of this file is ridiculous. Move out all the property retrieval / construction logic out so that only the UI part is left in this file.

Reply:
Moved out all property constructed relation functions to its own file.